### ISSUE TITLE
Unable to login to web interface with Windows authentication and Internal membership

### ISSUE THREAD

**Issue by brianhv** [jakubgarfield/Bonobo-Git-Server#758]
You may remember me from the excitement that was ticket #668.

Today I tried a fresh install of 6.3. Unlike before I've been able to get a clone working fairly quickly. However I haven't been able to log into the web interface when my `AuthenticationProvider` is `Windows`.

I started by logging in as admin while using `Cookies`. I created a user that matches my Windows username (and had to give it a password), and created a repo and made my new user an admin for it.

Then I changed my provider to `Windows`. At this point I was able to clone the repo using my Windows username/password. However, I could not log into the web interface. Interestingly, using my Windows password generated a  "The username or password provided is incorrect" error, but using the password I entered when creating the user just looped back to the login screen with no error. This suggests to me that even with the `Windows` auth provider, it's still checking against the internal password.

The log messages that get generated from the web interface:

```
2017-11-10 17:26:43.309 -05:00 [Verbose] EF: Validating user "brianhv"
2017-11-10 17:26:43.480 -05:00 [Verbose] EF: User "brianhv" validation result Failure
```

**Comment by larshg** [comment]
Please ses #755

**Comment by 2r0ck** [comment]
I have this problem.
This is due to a remote attribute [WebAuthorize] in Index() method( from HomeController  and RepositoryController) (look [710](https://github.com/jakubgarfield/Bonobo-Git-Server/pull/710) ).  I added this attribute and the service worked fine.

**Comment by willdean** [comment]
If I can make the time I'm about to release a 6.4.0 which removes the troublesome PR 710

**Comment by 2r0ck** [comment]
@willdean , thanks!  but the problem with anonymous access to the repositories will not be solved..

**Comment by willdean** [comment]
@2r0ck No, sure - the new feature will be gone for now, but at least it won't be broken for existing AD users who are currently being told to downgrade to 6.2.1
